apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import AnalyticsApp from './AnalyticsApp';
+import { render, screen } from '@testing-library/react';
+import runReportResponseHasViews from '../../../../../lambda/public/sampleData/runReportResponseHasViews.json';
```

Comment:
The thought behind the `/shared` folder was to work around imports like this, so by putting them in that folder you can access them here with `@shared/runReportResponseHasViews.json`, and the lambda could do the same. But right now these files are in a statically served `public` directory from express, so not sure the best way to handle this. My vote would be to also statically serve the `/shared` folder but that might break some node/express best practice. 

@ghepting @jsdalton thoughts?

Reply:
it's fine to import these in specs like this IMO. once the frontend isn't using them then we can just move them into a lambda/test/fixtures directory and this will be cleaner.